### DISCUSSION TITLE
Just don't!

### DISCUSSION THREAD

**Discussion by iagosousadev** [community/community#13833] [Other Feature Feedback, Questions, & Ideas]
I wanna enter github and see my work and do my usual other things either on my personal account or the other, the company one. I don't want to log into github and find that it has turned into a new mutant like social media where people come to post or to just show some random repos of unrelated things from what I do, github is (or was depending on how this goes) the only platform where I could come, and just look at my own things, work where I need to, see what I need to see.

The fact that I have to even try to explain how bad this is is already wrong in so many levels, github is not a fancy social media, it's a code versioning platform at its core, plus now also a place for open source communities to collaborate and where teams can develop things together. You don't need to mess with something that is already good, make your servers even better, make your processes and procedures even better, make the service that you provide even better, don't try to reinvent the wheel and change this into some fancy boring and toxic social media

All in all we know we can't make the final decision about the platform as it is, but whenever distraction becomes way too powerful to overcome then changes must be made, changing platforms is as easy as writing a hello world in python. I already close everything else to focus on my work on github and even here I will have to be handling cluttering because of a feed? It's already annoying enough that I get to see what my friends are starring or following, I don't need that it's dumb to force this. What about next? Gonna make everyone see whenever I push to a private or a company private repo? Gonna transform my whole work into a nightmare because now even on github I can't concentrate anymore?

That is just a horrible addition that we never asked for and whoever had this idea is just out of hid(her) mind.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
